test/unit/models/index.test.js

```diff
@@ -72,6 +66,12 @@ describe('test installation model', () => {
     });
   });
 
+  afterEach(async () => {
+    // Clean up the database
+    await Installation.truncate({ cascade: true, restartIdentity: true });
+    await Subscription.truncate({ cascade: true, restartIdentity: true });
```

Comment:
The only time I can see this being a problem is if the tests failed due to an exception and did not exit cleanly. This means the next run of the tests might have state from the previous run. Is there anything we can/should do about that?

Reply:
Even if the test fails, the `afterEach` is still called.  It's better to do it this way as it's the recommended testing approach for all testing frameworks - start with clean slate, end with clean slate.  

The problem with having it in the `beforeEach` is that when another test suite starts it will have some state already there, where the assumption is that there would be none.  If another suite installed a models that this one doesn't know about, you'll have to truncate that one also, leading to adding extra code just for cleaning up another suite.

Reply:
Yes, agreed on all counts, I meant in the case of a catastrophic exception (for example a ctrl-c). We probably don’t need to optimize for that case, but I wanted to raise the idea just in case.

Reply:
Right.  We were going to address that later when rewriting the tests so that it doesn't have a dependency on an external postgres service for them to run :)